sylt-parser/src/statement.rs

```diff
@@ -33,6 +33,15 @@ pub enum StatementKind {
         file: PathBuf,
     },
 
+    /// "Imports" variables from another file.
+    ///
+    /// `from <file> use <var1>
+    From {
+        path: Identifier,
+        imports: Vec<(Identifier, Option<Identifier>)>,
```

Comment:
`Option<Identifier>` looks a bit cryptic... Maybe make a type alias for this? Or use a struct, since they're often easier to read. Might not be super clear that the option is for renaming.

Reply:
it could also be an enum.

```rust
pub enum Import {
    Normal(Identifier),
    Renamed(Identifier, Identifier),
    // or
    Renamed {
        old: Identifier
        new: Identifier,
    }
}